controllers/Index.php

```diff
@@ -509,7 +509,11 @@ protected function makeObjectFormWidget($type, $object, $alias = null)
 
         $widgetConfig = $this->makeConfig($formConfigs[$type]);
         $widgetConfig->model = $object;
-        $widgetConfig->alias = $alias ?: 'form'.studly_case($type).md5($object->getFileName());
+        if (!$object->exists) {
+            $widgetConfig->alias = $alias ?: 'form' . studly_case($type) . uniqid();
+        } else {
+            $widgetConfig->alias = $alias ?: 'form' . studly_case($type) . md5($object->getFileName());
+        }
```

Comment:
Can you simplify this to match https://github.com/octobercms/october/commit/c23d671f9104f2239fc77c9601d48eb5c6db5c3a?

Reply:
But that one-liner makes my eye twitch :P

Reply:
I know, but it's how it was done in the main repo so I'd prefer it to match. And relatively speaking it's not that bad 😝

Reply:
@LukeTowers sorted. (*eyes twitch further*)

Reply:
You should uh, get that looked at 😆